flink-formats/flink-avro/src/test/java/org/apache/flink/formats/avro/AvroDeserializationSchemaTest.java

```diff
@@ -37,46 +38,56 @@ class AvroDeserializationSchemaTest {
 
     private static final Address address = TestDataGenerator.generateRandomAddress(new Random());
 
+    private static final AvroEncoding[] ENCODINGS = {AvroEncoding.BINARY, AvroEncoding.JSON};
+
     @Test
     void testNullRecord() throws Exception {
```

Comment:
Just a suggestion -- almost all of these tests could be run as JUnit parameterized tests.   For example:

```
    @ParameterizedTest
    @EnumSource(AvroEncoding.class)
    void testNullRecord(AvroEncoding encoding) throws Exception {
        DeserializationSchema<Address> deserializer = 
            AvroDeserializationSchema.forSpecific(Address.class, encoding);

        Address deserializedAddress = deserializer.deserialize(null);
        assertThat(deserializedAddress).isNull();
    }
```

This changes little when everything is working, but it adds additional context to the unit test when *one* of the AvroEncoding types fails.

Reply:
Nice - that's a new one on me, thanks. That's much better